api/src/test/java/uk/gov/dhsc/htbhf/claimant/message/processor/CompleteNewCardMessageProcessorTest.java

```diff
@@ -76,10 +76,9 @@ void shouldProcessCompleteNewCardMessage() {
 
     private void verifyMakeFirstPaymentMessageSent(Claim claim, PaymentCycle paymentCycle) {
```

Comment:
Rename test to verifyRequestFirstPaymentMessageSent

Reply:
Renamed.